Documentation~/en/audiostreaming.md

```diff
@@ -0,0 +1,61 @@
+# Audio Streaming

```

Comment:
This is under en, so should be written in English ?

Reply:
Yes, I will request Chris translating it in English